wsclient/src/main/java/org/hpccsystems/ws/client/antlr/Ecl.g4

```diff
@@ -0,0 +1,53 @@
+grammar Ecl;		
```

Comment:
Can we make this more specific? I think this grammar is describing the EclRecordDefinition?
 Also, the file should probably change the file name as well.